eng/common/scripts/Helpers/Service-Level-Readme-Automation-Helpers.ps1

```diff
@@ -11,8 +11,10 @@ function create-service-readme(
   $readmePath = Join-Path $readmeFolder -ChildPath $readmeName
   $content = ""
   if (Test-Path (Join-Path $readmeFolder -ChildPath $indexTableLink)) {
+    $escapedIndexTableLink = $indexTableLink -replace '\(', '\(' -replace '\)', '\)'
```

Comment:
What am I missing here? How does this escape anything? It looks like it is replacing with the same text.

Reply:
The syntax roughly amounts to this: 

``` 
"string" -replace 'regex', 'literalstring'
```

In `-replace` the left parameter is a regular expression so you have to escape the special parens character... the right parameter is a literal string and since we're in PowerShell we don't have to escape the backslash `\`.

Reply:
Added comments